### PR TITLE
Removes borders around form sections

### PR DESCRIPTION
**Pull request by daniellemaxwell**
### Scope of changes

This removes borders that appeared around smaller sections of the registration form. The borders were removed from the Legal Form, Contacts Form, and TRISA Implementation Form.

Below is an image of the Legal Form which displays the borders that have been removed in this PR:
https://www.awesomescreenshot.com/image/45230137?key=90d86e2a4ffbb0639a2196b56934f4b3

Note: Padding above the buttons will addressed in another story.

### Type of change

- [ ] bug fix
- [ ] new feature
- [ ] documentation
- [ ] other (describe)

### Acceptance criteria

Note: Some changes were made to allow me to make changes while not being able to log in. This caused the form to take longer to load in the video below, so no stories will be created to address the issues that appear when in the video.

https://www.awesomescreenshot.com/video/23806300?key=464e6853ac043ab69eb732284a0f981f

### Author checklist

- [x] I have manually tested the change and/or added automation in the form of unit tests or integration tests
- [ ]  I have updated the dependencies list
- [ ]  I have recompiled and included new protocol buffers to reflect changes I made
- [ ]  I have added new test fixtures as needed to support added tests
- [x]   Check this box if a reviewer can merge this pull request after approval (leave it unchecked if you want to do it yourself)
- [x]  I have moved the associated Shortcut story to "Ready for Review"

### Reviewer(s) checklist

- [ ] Any new user-facing content that has been added for this PR has been QA'ed to ensure correct grammar, spelling, and understandability.

### CODE REVIEW

File: web/gds-user-ui/src/components/Addresses/index.tsx [hunk 1]
```diff
@@ -1,12 +1,11 @@
-import { Heading, Text } from '@chakra-ui/react';
+import { Heading, Stack, Text } from '@chakra-ui/react';
 import { Trans } from '@lingui/react';
-import FormLayout from 'layouts/FormLayout';
 import React from 'react';
 import AddressList from './AddressList';
 
 const Addresses: React.FC = () => {
   return (
-    <FormLayout>
```
Comment (daniellemaxwell) [inline web/gds-user-ui/src/components/Addresses/index.tsx:9]: The `FormLayout` component has a styles set to display the border that we wanted to remove from around the form's smaller sections. The `FormLayout` component is used throughout the UI, so replacing it with the `Stack` component allows for the content to be grouped together. This change will also make it easier to apply specific styles to a smaller section within a form.

File: web/gds-user-ui/src/components/LegalPerson/LegalForm.tsx [hunk 2]
```diff
@@ -3,7 +3,7 @@ import { chakra, useDisclosure } from '@chakra-ui/react';
 import CountryOfRegistration from 'components/CountryOfRegistration';
 import FormLayout from 'layouts/FormLayout';
 import NameIdentifiers from '../NameIdentifiers';
-import NationalIdentification from '../NameIdentification';
+import NationalIdentification from '../NationalIdentification';
```
Comment (daniellemaxwell) [inline web/gds-user-ui/src/components/LegalPerson/LegalForm.tsx:6]: The file name was changed to make it easier to locate the component.

File: web/gds-user-ui/src/components/Contacts/ContactsForm.tsx [hunk 3]
```diff
@@ -148,21 +148,27 @@ const ContactsForm: React.FC<ContactsFormProps> = ({ data, shouldResetForm, onRe
             title={t`Legal/ Compliance Contact (required)`}
             description={t`Compliance officer or legal contact for requests about the compliance requirements and legal status of your organization. A business phone number is required to complete physical verification for MainNet registration. Please provide a phone number where the Legal/ Compliance contact can be contacted.`}
           />
-          <ContactForm
+          <Box pt={5}>
```
Comment (daniellemaxwell) [inline web/gds-user-ui/src/components/Contacts/ContactsForm.tsx:151]: `Box` is Chakra UI's (the component library used for this app) way of adding a `<div>`.

**Review by pdamodaran — APPROVED** [review]
approved @daniellemaxwell!

**Pull request merged**